command.go

```diff
@@ -64,6 +64,9 @@ type Command struct {
 	// render custom help text by setting this variable.
 	CustomHelpTemplate string
 
+	// Use longest prefix match for commands
+	AllowShorthandCommand bool
```

Comment:
Rename this one to `PrefixMatchCommands`, too?

Reply:
Done